src/node.cc

```diff
@@ -3589,6 +3589,10 @@ static void ParseArgs(int* argc,
   const char** new_v8_argv = new const char*[nargs];
   const char** new_argv = new const char*[nargs];
   const char** local_preload_modules = new const char*[nargs];
+#if HAVE_OPENSSL
+  bool use_bundled_ca = false;
+  bool use_openssl_ca = false;
+#endif  // HAVE_INSPECTOR
```

Comment:
Just noticed that the endif comment is off here. Also, this just adds the new variables… would they actually be necessary?